file.c

```diff
@@ -584,7 +584,7 @@ char *file_read_info_file(char *file, file_key_flag_t flags)
   file_t *f0 = NULL;
 
 #ifdef DEBUG_FILE
-  fprintf(stderr, "looking for info file: %s\n", file);
+  log_debug("looking for info file: %s\n", file);
```

Comment:
I do not get much why it needs to be closed in DEBUG_FILE directive? Is it some kind of mentioned info file?